source/upgrade-guide/upgrading-elastic-stack/elastic_server_hard_upgrade.rst

```diff
@@ -2,7 +2,7 @@
 
 .. _elastic_server_hard_upgrade:
 
-Upgrading Elastic Stack from 6.x to 6.8
+Upgrading Elastic Stack from 6.x to 6.x
```

Comment:
This must be 6.8. Elastic, in theory, will not release a 6.9, the 6.8 will be permanent and all changes on this version will be on patches.

Reply:
Done!